style.css

```diff
@@ -699,11 +799,27 @@ html {
     margin-left: 12px;
   }
 
-  .sBottomNav ul {
+  .sBottomNav {
+    justify-content: center;
+  }
+
+  .close-pop-up {
+    top: 38px;
+    right: 38px;
+  }
+
+  .second-div-pop-up {
```

Comment:
- [ ] On mobile, I can't see the close button because there is an overflow and the cards are not properly aligned, see from the images below.

![overflow](https://user-images.githubusercontent.com/53124548/136145111-dbf9f022-8d2f-4ea8-963e-6cc4a7658ccf.png)

![style](https://user-images.githubusercontent.com/53124548/136145120-876a24e4-f6d2-48ad-9f3d-85b019d4b151.png)